cmd/terramate/cli/cloud.go

```diff
@@ -768,6 +768,7 @@ func setGithubPRMetadata(md *cloud.DeploymentMetadata, pull *github.PullRequest)
 }
 
 func (c *cli) newReviewRequest(pull *github.PullRequest, reviews []*github.PullRequestReview, checks []*github.CheckRun, merged bool) *cloud.ReviewRequest {
+	pullUpdatedAt := pull.GetUpdatedAt()
```

Comment:
Could you remove the metadata sent the same information?

Reply:
If it's not delayed, I'm ok with it but sending same information twice is misleading and a waste of network resources :)

Reply:
I have added it to our backlog and will discuss with the team today.

Reply:
In this case, we should make it consistent and send always this information in the same way to cloud.